test/AElf.Contracts.MultiToken.Tests/MultiTokenContractWithCustomSystemTransactionTest.cs

```diff
@@ -66,7 +66,7 @@ public async Task TokenContract_WithSystemTransaction()
                         Amount = transferAmountInSystemTxn,
                         Memo = "transfer test",
                         Symbol = DefaultSymbol,
-                        To = Address.Zero
+                        To = AddressHelper.StringToAddress("to")
```

Comment:
make AddressHelper.StringToAddress("to") as a property in a helper method in a test base project.